### PR TITLE
Added a check to see if the revision number is '0' instead of '000'

### PR DESCRIPTION
**Pull request by RohitKandimalla**
MAT-2832
## Description

The revision number of Cql library is retrieved as '0' and is stored in measure XML, but the actual cql library will have '000' as the revision number. This mismatch is causing an issue while replacing the included libraries from helper UI. 
Solution: While retrieving the cql Library, if the revision number is '0' then we are returning '000' instead So that it could be saved as '000' in measure Xml.

## JIRA Ticket

[MAT-2832](https://jira.cms.gov/browse/MAT-2832)
## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)
- [ ] None applicable

### CODE REVIEW

File: src/main/java/mat/server/CQLLibraryService.java [hunk 1]
```diff
@@ -249,7 +249,7 @@ public CQLLibraryDataSetObject extractCQLLibraryDataObject(CQLLibrary cqlLibrary
         dataSetObject.setReleaseVersion(cqlLibrary.getReleaseVersion());
         dataSetObject.setQdmVersion(cqlLibrary.getQdmVersion());
         dataSetObject.setFhirVersion(cqlLibrary.getFhirVersion());
-        if (cqlLibrary.getRevisionNumber() == null) {
+        if (cqlLibrary.getRevisionNumber() == null || cqlLibrary.getRevisionNumber().equalsIgnoreCase("0")) {
```
Comment (RohitKandimalla) [inline src/main/java/mat/server/CQLLibraryService.java:252]: This is the fix.

**Review by jkotanchik-SB — APPROVED** [review]
